client/jetpack-cloud/sections/agency-dashboard/sites-overview/site-top-header-buttons/index.tsx

```diff
@@ -72,6 +76,26 @@ export default function SiteTopHeaderButtons() {
 							)
 						}
 					/>
+					{ shouldRenderAddSitesTour && (
+						<GuidedTour
+							className="jetpack-cloud-site-dashboard__guided-tour"
+							preferenceName="jetpack-cloud-site-dashboard-add-new-site-tour"
+							tours={ [
+								{
+									target: '#sites-overview-add-sites-button .split-button__toggle',
+									popoverPosition: 'bottom left',
+									title: translate( 'Press on an arrow button' ),
```

Comment:
```suggestion
									title: translate( 'Click the arrow button' ),
```

Reply:
We got that directly from the Figma. Should we change this despite what's specified in the Figma?

<img width="377" alt="image" src="https://github.com/Automattic/wp-calypso/assets/37049295/4204dd91-f8ce-4105-8ffb-135d94b90c11">

Reply:
This sounds better, but is redundant with the description, so probably should tweak the description.

Reply:
Yes, definitely needs a change.

Reply:
This change was added: 758e2d5b1d928f46831f8e38dfaa0e45e3eb72e6

Reply:
@coder-karen This doesn't appear to have been changed?

Reply:
No, I'm linking to a commit used for a comment below. @gogdzl I'll leave this one for you.

Reply:
Changed title! 1449c29